src/components/defectReport/DefectReportCreateForm.tsx

```diff
@@ -257,10 +257,10 @@ export const DefectReportCreateForm = ({
 
         <Controller
           name="termsOfService"
-          render={({ onChange, value }) => (
+          render={({ field }) => (
             <Checkbox
-              checked={!!value}
-              onPress={() => onChange(!value)}
+              checked={!!field.value}
+              onPress={() => field.onChange(!field.value)}
```

Comment:
In some parts of the code `{ field: { name, onChange, value }` is used in some parts. Can you edit the whole code in the same format?

Reply:
I've unified the code in this [commit](https://github.com/smart-village-solutions/smart-village-app-app/pull/901/commits/8753d75e2e49f895b382e945e241394c0a74dd4b)